src/test/ui/did_you_mean/bad-assoc-ty.stderr

```diff
@@ -46,7 +46,7 @@ error[E0223]: ambiguous associated type
 LL | type A = [u8; 4]::AssocTy;
    |          ^^^^^^^^^^^^^^^^ ambiguous associated type
    |
-   = note: specify the type using the syntax `<[u8; <unevaluated[]>] as Trait>::AssocTy`
```

Comment:
I did not fix this case because it would have been very involved. The error reporting mechanisms around this error are very string heavy. This would need to be addressed in a future PR